<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AVSoftTestCase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1480</width>
      <height>550</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1480</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1480</width>
      <height>550</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollWidgetTree">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1478</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAVSoftTestCase">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="closeFile"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="separator"/>
    <addaction name="redo"/>
    <addaction name="undo"/>
   </widget>
   <addaction name="menuAVSoftTestCase"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFile">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="redo">
   <property name="text">
    <string>Вперед... (Ctrl + Y)</string>
   </property>
  </action>
  <action name="undo">
   <property name="text">
    <string>Назад... (Ctrl + Z)</string>
   </property>
  </action>
  <action name="closeFile">
   <property name="text">
    <string>Закрыть...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
